resources/views/components/claim-new-table-row.blade.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+use Illuminate\Support\Carbon;
+
+$startedTimeAgo = Carbon::createFromFormat("Y-m-d H:i:s", $claim['DoneTime'])->diffForHumans();
```

Comment:
This should use the claim created date, not the expiration date.

Reply:
🤦‍♂️ D'oh. Addressed in https://github.com/RetroAchievements/RAWeb/pull/1425/commits/c2825edbcf8c4be73c349a67619991b666c00390.